BookingsApi/BookingsApi.IntegrationTests/Api/V2/Hearings/UpdateHearingsInGroupV2Tests.cs

```diff
@@ -569,88 +571,100 @@ private void AssertEventsPublished(Hearing hearing, HearingRequestV2 requestHear
             }
-            
             void AssertJudiciaryParticipantEvents()
             {
                 var expectedAddedCount = requestHearing.JudiciaryParticipants.NewJudiciaryParticipants.Count;
-            
                 var participantAddedMessages = messages
                     .Where(x => x.IntegrationEvent is ParticipantsAddedIntegrationEvent)
                     .Select(x => x.IntegrationEvent as ParticipantsAddedIntegrationEvent)
                     .Where(x => x.Hearing.HearingId == hearing.Id)
                     .ToList();
-
                 participantAddedMessages.Count.Should().Be(expectedAddedCount);
-
                 var expectedRemovedCount = requestHearing.JudiciaryParticipants.RemovedJudiciaryParticipantPersonalCodes.Count;
-            
                 var participantRemovedMessages = messages
                     .Where(x => x.IntegrationEvent is ParticipantRemovedIntegrationEvent)
                     .Select(x => x.IntegrationEvent as ParticipantRemovedIntegrationEvent)
                     .Where(x => x.HearingId == hearing.Id)
                     .ToList();
-
                 participantRemovedMessages.Count.Should().Be(expectedRemovedCount);
             }
-
             void AssertEndpointEvents()
             {
                 var expectedAddedCount = requestHearing.Endpoints.NewEndpoints.Count;
-
                 var endpointAddedMessages = messages
                     .Where(x => x.IntegrationEvent is EndpointAddedIntegrationEvent)
                     .Select(x => x.IntegrationEvent as EndpointAddedIntegrationEvent)
                     .Where(x => x.HearingId == hearing.Id)
                     .ToList();
-
                 endpointAddedMessages.Count.Should().Be(expectedAddedCount);
-                
                 var expectedUpdatedCount = requestHearing.Endpoints.ExistingEndpoints.Count;
-
                 var endpointUpdatedMessages = messages
                     .Where(x => x.IntegrationEvent is EndpointUpdatedIntegrationEvent)
                     .Select(x => x.IntegrationEvent as EndpointUpdatedIntegrationEvent)
                     .Where(x => x.HearingId == hearing.Id)
                     .ToList();
-
                 endpointUpdatedMessages.Count.Should().Be(expectedUpdatedCount);
-                
                 var expectedRemovedCount = requestHearing.Endpoints.RemovedEndpointIds.Count;
-                
                 var endpointRemovedMessages = messages
                     .Where(x => x.IntegrationEvent is EndpointRemovedIntegrationEvent)
                     .Select(x => x.IntegrationEvent as EndpointRemovedIntegrationEvent)
                     .Where(x => x.HearingId == hearing.Id)
                     .ToList();
-
                 endpointRemovedMessages.Count.Should().Be(expectedRemovedCount);
             }
         }
+        
+        private void AssertNotificationEvents(Hearing hearing, HearingRequestV2 requestHearing, int existingParticipantsUpdated)
```

Comment:
`requestHearing` and `existingParticipantsUpdated` are not used

Reply:
done